app/views/content/returning-to-teaching.md

```diff
@@ -165,6 +155,7 @@ If you’re returning to teaching, you might want to re-familiarise yourself wit
 #### Other things to consider
 
 * attend an [online return to teaching event](/event-categories/online-q-as)
+* return to the classroom by [registering with a supply teaching agency](/urgent-call-for-qualified-teachers)
```

Comment:
Thanks for changing this @tcannon3 - could you move it to be the last bullet point as well? Thanks!

Reply:
Oops sorry, thought I had. Will move it down now